player.go

```diff
@@ -69,26 +69,25 @@ func (a *API) GetDevices() ([]*Device, error) {
 
 // Play starts a new context or resume current playback on the user's active device.
 // https://developer.spotify.com/documentation/web-api/reference/#endpoint-start-a-users-playback
-func (a *API) Play(deviceID string, uris ...string) error {
+// contextURI added to allow for album and playlist support - A blank context uri will play just the track uri.
+func (a *API) Play(deviceID, contextURI string, uris ...string) error {
 	v := make(url.Values)
 	if deviceID != "" {
 		v.Add("device_id", deviceID)
 	}
 
-	if len(uris) == 0 {
-		return a.put("v1", "/me/player/play", v, nil)
-	}
-
 	body := &struct {
-		URIs []string `json:"uris"`
-	}{URIs: uris}
+		ContextURIs string `json:"context_uri,omitempty"`
+		URIs []string `json:"uris,omitempty"`
+	}{ContextURIs: contextURI,URIs: uris}
```

Comment:
Run `go fmt` on this file to fix the bad formatting here... I'll set up `golangci-lint` for this project soon.